src/main/java/com/ezasm/DiscordActivity.java

```diff
@@ -0,0 +1,72 @@
+package com.ezasm;
+
+import java.io.File;
+import java.time.Instant;
+
+import de.jcm.discordgamesdk.Core;
+import de.jcm.discordgamesdk.CreateParams;
+import de.jcm.discordgamesdk.activity.Activity;
+
+public final class DiscordActivity {
+
+    private static final long discordApplicationId = 1068652802483691641L;
+    private static final Activity activity;
+    private static final Core core;
+
+    static {
+        Core.init(new File("discord_game_sdk.so"));
+
+        final var params = new CreateParams();
+        params.setClientID(discordApplicationId);
+
+        core = new Core(params);
+
+        activity = new Activity();
+        activity.setDetails("I'm using EzASM");
+        activity.setState("EzASM is easy");
+        activity.timestamps().setStart(Instant.now());
+
+        updateActivity();
+    }
+
+    /**
+     * This should be called by the main thread.
+     */
+    public static void runCore() {
+        while (true) {
+            core.runCallbacks();
+            try {
+                Thread.sleep(16);
```

Comment:
use ```LockSupport.parkNanos``` instead. Also why 16ms?

Reply:
It was the recommended interval of time that I saw on a code example of the library.